CMSgov/easi-app
# [EASI-2991] Admin home view switcher

# EASI-2991

## Changes and Description

- Select dropdown on home page of TRB/IT Gov if user has both job codes that renders the respective homepage
  - Defaults to TRB
- Selection is persisted to local storage
- Unit tests



## How to test this change

- Log in as user with both `EASI_D_GOVTEAM` and `EASI_TRB_ADMIN_D` job codes
- Ensure dropdown field renders, works as expected, and persists selection to local storage

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
